src/Chapter3.hs

```diff
@@ -459,6 +480,14 @@ and provide more flexibility when working with data types.
 Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
+data HobbitMeals
+  = Breakfast
+  | SecondBreakfast
+  | Elevenses
+  | Luncheon
+  | AfternoonTea
+  | Dinner
+  | Supper
```

Comment:
So cute :blush: :heart_eyes:

Reply:
Hobbits have the largest number of meals I could think of :joy: